src/platform/apollolake/include/platform/memory.h

```diff
@@ -270,6 +270,24 @@
 #define HP_SRAM_WIN3_BASE	SRAM_TRACE_BASE
 #define HP_SRAM_WIN3_SIZE	SRAM_TRACE_SIZE
 
+#if defined(CONFIG_DMIC)
+#define SOF_TEXT_DMIC_SIZE 0x2000
```

Comment:
We can accept this for the moment, but ideally we should now look at using a linker variable to set SOF_TEXT_BASE, etc for heap positioning.